crates/sui-framework/packages/move-stdlib/sources/string.move

```diff
@@ -105,4 +119,14 @@ module std::string {
     native fun internal_is_char_boundary(v: &vector<u8>, i: u64): bool;
     native fun internal_sub_string(v: &vector<u8>, i: u64, j: u64): vector<u8>;
     native fun internal_index_of(v: &vector<u8>, r: &vector<u8>): u64;
+
+    // === Deprecated ===
+
+    /// [DEPRECATED]
+    public fun bytes(s: &String): &vector<u8> { s.as_bytes() }
+
+    /// [DEPRECATED]
+    public fun sub_string(s: &String, i: u64, j: u64): String {
+        s.substring(i, j)
+    }
```

Comment:
@tzakian

Reply:
@damirka `#[deprecated]` should land soon ™️ , if it hasn't already

Reply:
Literally just landed!

Reply:
And.... it's gone